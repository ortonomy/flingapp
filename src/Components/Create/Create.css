.sidebar-link {
  color: #4990e2;
  text-transform: uppercase;
  font-size: 0.9em;
  font-weight: bold;
  padding-top: 2em;
}
