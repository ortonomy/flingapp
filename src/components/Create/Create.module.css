.Create {
  composes: paddedBox from '../Sidebar/Sidebar.module.css';
}

.link {
  color: #4990e2;
  text-transform: uppercase;
  font-size: 0.9em;
  font-weight: bold;
  padding-top: 2em;
}

.sectionTitle {
  composes: sectionTitle from '../Sidebar/Sidebar.module.css';
}
