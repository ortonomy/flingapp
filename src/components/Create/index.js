import React, { Component } from 'react';
import styles from './Create.module.css';


class Create extends Component {
  render() {
    return (
      <div className={styles.Create}>
        <div className={styles.sectionTitle}>
          <i className='fa fa-plus-circle'></i>
          <span>Create</span>
         </div>
         <div className={styles.link}>
            New Freelancer
         </div>
         <div className={styles.link}>
            New Project
         </div>
      </div>
    )
  }
}

export default Create;
