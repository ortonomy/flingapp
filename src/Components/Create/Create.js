import React, { Component } from 'react';
import './Create.css';


class Create extends Component {
  render() {
    return (
      <div className="Create">
        <div className='sidebar-section-title'>
          <i className='fa fa-plus-circle'></i>
          <span>Create</span>
         </div>
         <div className='sidebar-link'>
            New Freelancer
         </div>
         <div className='sidebar-link'>
            New Project
         </div>
      </div>
    )
  }
}

export default Create;
