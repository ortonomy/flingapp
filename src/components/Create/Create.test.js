import React from 'react';
import Create from '../Create';

it('renders without crashing', () => {
  const wrapper = shallow(<Create />);
  expect(wrapper).toMatchSnapshot();
});
