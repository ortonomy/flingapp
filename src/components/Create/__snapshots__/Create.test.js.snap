// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Create />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className={undefined}
>
          <i
                    className="fa fa-plus-circle"
          />
          <span>
                    Create
          </span>
</div>,
        <div
          className={undefined}
>
          New Freelancer
</div>,
        <div
          className={undefined}
>
          New Project
</div>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <i
              className="fa fa-plus-circle"
/>,
            <span>
              Create
</span>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-plus-circle",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Create",
            },
            "ref": null,
            "rendered": "Create",
            "type": "span",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "New Freelancer",
          "className": undefined,
        },
        "ref": null,
        "rendered": "New Freelancer",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "New Project",
          "className": undefined,
        },
        "ref": null,
        "rendered": "New Project",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className={undefined}
>
            <i
                        className="fa fa-plus-circle"
            />
            <span>
                        Create
            </span>
</div>,
          <div
            className={undefined}
>
            New Freelancer
</div>,
          <div
            className={undefined}
>
            New Project
</div>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <i
                className="fa fa-plus-circle"
/>,
              <span>
                Create
</span>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-plus-circle",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Create",
              },
              "ref": null,
              "rendered": "Create",
              "type": "span",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "New Freelancer",
            "className": undefined,
          },
          "ref": null,
          "rendered": "New Freelancer",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "New Project",
            "className": undefined,
          },
          "ref": null,
          "rendered": "New Project",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
